.github/workflows/release.yaml

```diff
@@ -0,0 +1,47 @@
+on:
+  release:
+    types: [published]
+
+name: Upload Release Asset
+
+jobs:
+  build:
+    name: Upload Release Asset
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Get release
+        id: get_release
+        uses: bruceadams/get-release@v1.2.2
+        env:
+          GITHUB_TOKEN: ${{ secrets.ACTIONS_TOKEN }}
+
+      - name: Publish tekton images and generate release manifests
+        run: |
+          echo ${{ secrets.QUAY_PASSWORD }} | podman login -u="${{ secrets.QUAY_BOT }}" --password-stdin quay.io
```

Comment:
Would `podman login -u "${{ secrets.QUAY_BOT }}" -p "${{ secrets.QUAY_PASSWORD }}" quay.io` work as well?

Reply:
yes, but it is insecure